both/access_control.js

```diff
@@ -258,38 +257,24 @@ if(Meteor.isServer) {
         'updatedAt',
         'createdAt',
       ];
-      return _onlyAllowedFields(fields, allowedFields);
+      return onlyAllowedFields(fields, allowedFields);
     },
     fetch: [ 'competitionId' ],
   });
 
   Registrations.allow({
     insert: function(userId, registration) {
-      return Registrations.userIsAllowed(userId, registration);
+      return Registrations.allowOperation(userId, registration, []);
```

Comment:
We actually support Registrations without Users (when importing data, we create Registrations, but not Users), so the normalization scheme you suggested is a non-starter.

For now, I like the idea of allowing people to edit these fields as long as registration is open. We can think more carefully about what (if anything) we want to allow people to edit after Registration is closed.